docs/sources/developers/plugins/build-a-streaming-data-source.md

```diff
@@ -0,0 +1,132 @@
++++
+title = "Build a streaming data source"
+type = "docs"
++++
+
+# Build a streaming data source
+
+> This guide assumes that you're already familiar with how to [Build a data source plugin]({{< relref "../../../../../tutorials/build-a-data-source-plugin.md" >}}).
```

Comment:
I recommend putting notes like this after the first paragraph. The first paragraph is often displayed in search results, so we want to have something that will help users know what the topic is about.